site/content/blog/2022-08-31-zap-updates-2022-august/index.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "ZAP Updates 2022 August"
+summary: >
+  All of the things that have been happening related to ZAP in August 2022.
+type: post
+tags:
+- blog
+  update
+date: "2022-08-31"
+authors: 
+    - simon
+---
+
+Lots of things are going on with ZAP but I've realised that many of you will not be aware of all of them.
+
+So heres a summary of what has been going on this month - let us know if you find this useful, 
+if it is then this could become a regular feature :grin:.
+
+### ZAP Version 2.12.0 - Coming Soon
+
+On 19th August we [announced](https://groups.google.com/g/zaproxy-users/c/2HTkM5fXTEg/m/9HstqC_fAAAJ) 
+that the next major version of ZAP, 2.12.0, will be "coming soon".
+There's no exact date yet, but most of the changes are already in the weekly releases, so download the most recent one if you want to see what it wil look like.
+
+### New Add-on: Param Digger
+
+This month saw the first release and associated [blog post](/blog/2022-08-22-the-param-digger-addon/) 
+for the [Param Digger](/docs/desktop/addons/parameter-digger/) add-on.
+This is being developed by [Arkaprabha Chakraborty](https://twitter.com/FValkyrie_17) as part of Google Summer of Code 2022.
+
+### Stats
+
+Statistics from the 2 main ZAP repos.
+
+TODO - these will need to be updated at the last possible time ;)
+
+#### [zaproxy](https://github.com/zaproxy/zaproxy/pulse/monthly)
+
+Excluding merges, 7 authors have pushed 19 commits to main and 19 commits to all branches. 
+On main, 327 files have changed and there have been 3,444 additions and 1,050 deletions. 
+
+#### [zap-extensions](https://github.com/zaproxy/zap-extensions/pulse/monthly)
+
+Excluding merges, 10 authors have pushed 67 commits to main and 67 commits to all branches. 
+On main, 1,277 files have changed and there have been 21,550 additions and 2,391 deletions. 
+
+### Ongoing Work
+
+#### Networking
+
+Work to upgrade the ZAP networking stack to support modern protocols is ongoing.
+The next phase will add initial support for HTTP/2 - this is a significant change and we expect this to be merged after
+2.12.0 has been released.
+
+#### Spider Migration to an Add-on
```

Comment:
The content under this section could include a link to the spider blog post once that's published.